cmd/docker-machine-driver-kvm/Makefile

```
@@ -1,4 +1,4 @@
 default: build
 
 build:
-	GOGC=off go build -i -o docker-machine-driver-kvm
+	GOGC=off go build -tags libvirt.1.2.14 -i -o docker-machine-driver-kvm
```

Unfortunately I don't have a physical ubuntu 14.04 box (or other older LTS distro) to try this out at the moment.  Was this alone sufficient to have this all hold together where it both compiles and doesn't do something nasty at runtime when the underlying API isn't present?  Maybe enough time has passed that we can ignore the older LTS distros, but I was hoping to keep this compatible for a while if possible.

---

I don't have access to such a machine. However latest Ubuntu LTS (xenial 16.04LTS) has [version 1.3.1 available](http://packages.ubuntu.com/search?keywords=libvirt&searchon=names&suite=xenial-updates&section=all).

Personally I would ignore 14.04.

---

I've been meaning to improve the build a bit... and this helped motivate me to get some extra build support in place to generate binaries for various host systems. https://github.com/dhiltgen/docker-machine-kvm/commit/f328c4b81d36269845812a6bd0046e64a9c7d877

I think we should be able to at least partially answer the compatibility question on this PR by seeing if it builds on ubuntu14.04.  Then we could try to do a create in a container, which should fail due to the missing libvirt socket, but as long as it errors that way and not with some obscure C lib link problem, then I think we're good.

---

I rebased my changes against your master, this is the error I get when I try to build the code inside of 14.04:

```
# github.com/dhiltgen/docker-machine-kvm
./kvm.go:596: cannot use uint(libvirt.DOMAIN_INTERFACE_ADDRESSES_SRC_LEASE) (type uint) as type libvirt.DomainInterfaceAddressesSource in argument to d.VM.ListAllInterfaceAddresses
```

Do you think it's acceptable?